### ISSUE TITLE
exit confirmation

### ISSUE THREAD

**Issue by maxicaleca** [dicksonleong/GagBook#8]
Is there any possiblity to add an "exit confirmation button"?

Thanks in advance, Maximiliano

**Comment by dicksonleong** [comment]
It is possible, but why do you think it is necessary?

**Comment by jara001** [comment]
I think I know the answer - sometimes you just hit that exit button by mistake.... :)

**Comment by maxicaleca** [comment]
Yes, and sometimes I 'm just looking pictures and I think I touch it and have to go back to continue looking other ones, and I exit the programme :S

**Comment by dicksonleong** [comment]
Ok, I will consider about this.
